src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -357,6 +357,20 @@ private function validateAndDenormalize(string $currentClass, string $attribute,
         throw new NotNormalizableValueException(sprintf('The type of the "%s" attribute for class "%s" must be one of "%s" ("%s" given).', $attribute, $currentClass, implode('", "', array_keys($expectedTypes)), gettype($data)));
     }
 
+    /**
+     * Get the types for the given class and attribute.
```

Comment:
`Gets`, but I think you can remove the description as it does not add anything that is not already part of the signature.

Reply:
Fair enough; removed!